indica/utilities.py

```diff
@@ -20,6 +20,7 @@
 from xarray.core.variable import Variable
```

Comment:
Most of the added functions here seem useful (maybe with the exception of `print_like` and `get_function_name`) but it would be helpful to include docstrings that are consistent with the `numpy` style for all of the added functions. Also unit tests for some of the more complicated functions like `intersection`.